### PR TITLE
Maximum

### LINKED ISSUE #17840: maximum

### PR DESCRIPTION
**Pull request by Arwa45**
close #17840

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by HaiderSultanArc** [comment]
```
@handle_frontend_method(
    class_tree=CLASS_TREE,
    init_tree="paddle.to_tensor",
    method_name="fmax",
    dtypes_and_x=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"), num_arrays=2, shared_dtype=True
    ),
)
def test_paddle_fmax(
    dtypes_and_x,
    frontend_method_data,
    init_flags,
    method_flags,
    frontend,
    on_device,
):
    input_dtype, x = dtypes_and_x
    helpers.test_frontend_method(
        init_input_dtypes=input_dtype,
        init_all_as_kwargs_np={"data": x[0]},
        method_input_dtypes=input_dtype,
        method_all_as_kwargs_np={"y": x[1]},
        frontend_method_data=frontend_method_data,
        init_flags=init_flags,
        method_flags=method_flags,
        frontend=frontend,
        on_device=on_device,
    )


@handle_frontend_method(
    class_tree=CLASS_TREE,
    init_tree="paddle.to_tensor",
    method_name="fmin",
    dtypes_and_x=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"), num_arrays=2, shared_dtype=True
    ),
)
def test_paddle_fmin(
    dtypes_and_x,
    frontend_method_data,
    init_flags,
    method_flags,
    frontend,
    on_device,
):
    input_dtype, x = dtypes_and_x
    helpers.test_frontend_method(
        init_input_dtypes=input_dtype,
        init_all_as_kwargs_np={"data": x[0]},
        method_input_dtypes=input_dtype,
        method_all_as_kwargs_np={"y": x[1]},
        frontend_method_data=frontend_method_data,
        init_flags=init_flags,
        method_flags=method_flags,
        frontend=frontend,
        on_device=on_device,
    )


@handle_frontend_method(
    class_tree=CLASS_TREE,
    init_tree="paddle.to_tensor",
    method_name="minimum",
    dtype_and_x=helpers.dtype_and_values(
        available_dtypes=helpers.get_dtypes("float"),
        num_arrays=2,
        shared_dtype=True,
    ),
)
def test_paddle_minimum(
    dtype_and_x,
    frontend_method_data,
    init_flags,
    method_flags,
    frontend,
    on_device,
):
    input_dtype, x = dtype_and_x
    helpers.test_frontend_method(
        init_input_dtypes=input_dtype,
        init_all_as_kwargs_np={"data": x[0]},
        method_input_dtypes=input_dtype,
        method_all_as_kwargs_np={"y": x[1]},
        frontend_method_data=frontend_method_data,
        init_flags=init_flags,
        method_flags=method_flags,
        frontend=frontend,
        on_device=on_device,
    )
```

Add this code below your test function in test file to resolve conflict.

**Comment by bipinKrishnan** [comment]
Hey @Arwa45, can you merge your branch with master. Seems like the tests you added are not picked up by the CI. Thanks :slightly_smiling_face:

**Comment by Arwa45** [comment]
> Hey @Arwa45, can you merge your branch with ivy master. Seems like the tests you added are not picked up by the CI. Thanks 🙂

hey @bipinKrishnan Do you want to merge ivy's branch with mine?
and why CI didn't pick the tests. It's on GitHub I can see that tests ran

**Comment by bipinKrishnan** [comment]
Hi @Arwa45 , the tests ran but it did not pick the test you added `test_paddle_maximum`. Thanks :slightly_smiling_face:

**Comment by HaiderSultanArc** [comment]
> Hey @Arwa45, can you merge your branch with ivy master. Seems like the tests you added are not picked up by the CI. Thanks 🙂

Hi @bipinKrishnan , How can @Arwa45 merge her branch with Ivy's master branch. Applicants don't get that kind of access. Reviewers have to merge the branch into Ivy's master branch.

**Comment by bipinKrishnan** [comment]
I meant to to merge ivy master onto their branch. Sorry for the confusion there :slightly_smiling_face:

**Comment by bipinKrishnan** [comment]
The tests are still failing due to some unexpected argument issue, fixing this should allow us to run the tests and see if the implementation is correct @Arwa45

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py [hunk 1]
```diff
@@ -1840,6 +1841,41 @@ def test_paddle_instance_rad2deg(
...
+    class_tree=CLASS_TREE,
+    init_tree="paddle.to_tensor",
+    method_name="maximum",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=2,
+        shared_dtype=True,
+    ),
+)
+def test_paddle_maximum(
```
Comment (bipinKrishnan) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1855]: This should be `test_paddle_instance_maximum`

Comment (Arwa45) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_tensor.py:1855]: done

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py [hunk 2]
```diff
@@ -1,8 +1,4 @@
-# global
-import numpy as np
```
Comment (bipinKrishnan) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2]: Can you bring these imports back as they are not related to your changes?

Comment (Arwa45) [reply ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_tensor.py:2]: done

**Pull request merged**